src/scripts/config_parser.py

```diff
@@ -0,0 +1,44 @@
+import os
```

Comment:
How about declaring the config structure via json schema (and adding inline requirements to the script to do so).

I think in this case, rather than parsing the file with dedicated parser, we could add validaton in the schema and convert the .yml file into outputs in a generic way:

```
x:
   y:
     z
```

=> `outputs.x.y.z`

Reply:
Sure thats possible we can do that, and thats easy to extract output aswell :) in other steps

Reply:
Believe one more check in svn  to validate the number file count per package? 
eg: our repo has

```
The following files should be present (6 files):

.tar.gz + .asc + .sha512 (one set of files per provider)
-py3-none-any.whl + .asc + .sha512 (one set of files per provider)
```

Reply:
* Config looks good!
* I'd say we should just check if each file has .asc and .sha512 - we do not need to fully verify the number of files, just making sure that all of them are signed and checksumed and that they are right - should be enough, so likely what we have in "checksum" and "signature" check will be enough.

Reply:
Cool :) will use same config then and use Json to validate schema.